src/test/java/org/folio/rest/impl/GOBIIntegrationServiceResourceImplTest.java

```diff
@@ -592,13 +664,24 @@ private void handleGetVendor(RoutingContext ctx) {
     private void handleGetMaterialType(RoutingContext ctx) {
 
       logger.info("got material-type request: {}", ctx.request().query());
+      JsonObject mtypes = new JsonObject();
 
-      JsonObject mtypes = new JsonObject()
-        .put("mtypes", new JsonArray()
+      if (ctx.request().query().contains("NonExistent")) {
+        mtypes.put("mtypes", new JsonArray())
+          .put("total_records", 0);
+      } else if (ctx.request().query().contains("unspecified")) {
```

Comment:
I'm a little surprised this is entirely lowercase, but I checked and indeed that's what's in inventory-storage